.github/workflows/oci.yml

```diff
@@ -42,12 +42,18 @@ jobs:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Login to Clinia Harbor
+        uses: docker/login-action@v3
+        with:
+          registry: oci.clinia.dev
+          username: ${{ secrets.CLINIA_HARBOR_USER }}
+          password: ${{ secrets.CLINIA_HARBOR_PASSWORD }}
       - name: Build and push by digest
         id: build
         uses: docker/build-push-action@v5
         with:
           context: .
-          file: ./Dockerfile
+          file: Dockerfile
```

Comment:
That's how its done in `atlas` so I'm assuming the `./` is redundant.

Reply:
J'ai même l'impression que c'est la valeur par défaut! (`PATH/Dockerfile`) 🙂 

This github action: https://github.com/docker/build-push-action
Which uses https://docs.docker.com/reference/cli/docker/buildx/build/#file